docs/fleet/configure-a-fleet.md

```diff
@@ -33,6 +33,13 @@ You must be an [organization owner](/fleet/rbac/#permissions) in order to create
 A fragment can define one, several, or all resources on a machine.
 You can add multiple fragments to a single machine, and you can add additional resources to a machine that has already been configured with a fragment.
 
+{{% alert title="Support Notice" color="note" %}}
+
+You cannot configure the [Viam Agent](/build/provision/) within a fragment.
+(You can, however, add a fragment inside of your Viam Agent config.)
```

Comment:
Nit: trying a more direct advisement, what do you think? I'd like to avoid the parens, and to link directly to the section that deals with configuring frag deployment via the Agent, if you agree:

```suggestion
While you can [use the Viam Agent to deploy a fragment](/build/provision/#use-a-provisioning-configuration-file) to your machine or fleet, you cannot configure the Viam Agent itself using fragments.
For more information, see [Viam Agent](/build/provision/).
```

Reply:
This is great TYSM

Reply:
I think the second sentence can come out so we don't link to the same thing twice inside it though